modules/system/classes/ImageResizer.php

```
@@ -912,27 +969,186 @@ public static function filterGetUrl($image, $width = null, $height = null, $opti
      */
     public static function filterGetDimensions($image): array
     {
-        $resizer = new static($image);
+        try {
+            $resizer = new static($image);
```

PHPStan (2.2.7)

[error] Unsafe usage of new static().
See: https://phpstan.org/blog/solving-phpstan-error-unsafe-usage-of-new-static

(new.static)